central/sensor/service/connection/connection_impl.go

```diff
@@ -366,6 +368,47 @@ func (c *sensorConnection) processIssueLocalScannerCertsRequest(ctx context.Cont
 	return nil
 }
 
+func (c *sensorConnection) processIssueSecuredClusterCertsRequest(ctx context.Context, request *central.IssueSecuredClusterCertsRequest) error {
+	requestID := request.GetRequestId()
+	clusterID := c.clusterID
```

Comment:
Is this always guaranteed to be filled in at the time this RPC is called?
Wondering about unusual ordering of the messages.

Reply:
In this case, I think gRPC will use the empty value, so there's no risk of a nil pointer dereference. The `requestId` is already being checked a few lines below, and the `clusterId` and `namespace` values are validated in other parts of the code. The [tests](https://github.com/stackrox/stackrox/pull/12740/files#diff-b7db23ca767f09df25045206d18c4ac8a30686615e05d4e5beb7278e7cea286eR454) also verify this.

But if you think it would be better to add explicit checks for `clusterId` and `namespace` within this function for added clarity, I can do that. I can also adjust `processIssueLocalScannerCertsRequest` since the logic is similar.